go.mod

```diff
@@ -27,10 +27,9 @@ require (
 	github.com/cosmos/gogoproto v1.7.0
 	github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v10 v10.0.0-beta.2
 	github.com/cosmos/ibc-apps/modules/rate-limiting/v10 v10.0.0-beta.2-bugfix.1
-	github.com/cosmos/ibc-go/modules/apps/callbacks/v10 v10.0.0-rc.2
-	github.com/cosmos/ibc-go/modules/light-clients/08-wasm/v10 v10.0.0-rc.2
-	github.com/cosmos/ibc-go/v10 v10.0.0
-	github.com/cosmos/interchain-security/v7 v7.0.0-rc0
+	github.com/cosmos/ibc-go/modules/light-clients/08-wasm/v10 v10.0.0-rc.3
+	github.com/cosmos/ibc-go/v10 v10.0.0-rc.3
+	github.com/cosmos/interchain-security/v7 v7.0.0-20250303152412-14fe074d9d95
```

Comment:
Worth noting that this is from the newly merged bump in interchain-security. After it bumps, this should be replaced